### PR TITLE
Create african-task-force-update.md

### PR DESCRIPTION
**Pull request by anelda**

### CODE REVIEW

**Comment by maneesha** [comment]
@ErinBecker   do you want to review this before merging?

File: _posts/2019/2019-01-11-african-task-force-update.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+- Developing a clear process for joining the task force and providing opportunities for the community 
+to meet with the task force can help to avoid exclusivity. The wider community should not get the idea 
+that one has to be part of the task force to be part of the Carpentries or to take initiative around activities 
+related to the task force goals.
+- Africa is a very large and diverse region and it is difficult to maintain an active, collaborative and
+inclusive task force made up of stakeholders living in vastly different places, with different technical resources.
+- Things that work for Carpentries communities elsewhere in the world, might work differently for a task force in Africa.
+Initiating and developing a task force in Africa has required (and is still requiring) re-thinking and re-structuring 
+within context.
+- Goals change as community needs arise and as the community matures (which doesn't happen uniformly accross regions, countries
```
Comment (ErinBecker) [inline _posts/2019/2019-01-11-african-task-force-update.md]: typo accross

File: _posts/2019/2019-01-11-african-task-force-update.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+reaching the goal. The goal post may also shift over time if it isn’t set clearly at the beginning. The instructor
+development committee discuss setting 
+[SMART goals on their mentoring groups page](https://github.com/carpentries/instructor-development/blob/master/mentoring-groups/program-outline.md). 
+
+## Communication
+
+The task force had a Google Group where we communicated and also met online using Vidyo. We were fortunate enough
+to have a large number of task force members together in South Africa in February and again in September and also
+met in person at CarpentryCon in Dublin, thanks to The Carpentries and other funders. The meetings were mostly held
+ad hoc rather than regularly scheduled meetings. A github repository within The Carpentries organisation account was
```
Comment (ErinBecker) [inline _posts/2019/2019-01-11-african-task-force-update.md]: GitHub

File: _posts/2019/2019-01-11-african-task-force-update.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+also made available to keep record of the minutes. 
+
```
Comment (ErinBecker) [inline _posts/2019/2019-01-11-african-task-force-update.md]: link to minutes

File: _posts/2019/2019-01-11-african-task-force-update.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+- running a monthly online meeting for African instructors in a regular time slot;
+- running a six-week online Carpentries workshop (Data Carpentry for Ecologists with R) pitched at trained 
+instructors with the intention to show them how the various lessons were taught and also assist them to 
+solidify their skills with the various tools; and
+- assisting trained instructors with the steps to check out.
+
+At the inception of the African Task Force, the instructor development committee did not yet exist and the
+[Carpentries' formal mentoring programme](https://carpentries.org/community/#mentoring) was not yet available. 
+Our activities focussed on some aspects that have since been taken over by the mentoring programme, 
+which means African instructors can now join the international community to receive mentorship.
```
Comment (ErinBecker) [inline _posts/2019/2019-01-11-african-task-force-update.md]: link to how to join? community discussions Etherpad?

**Comment by ErinBecker** [comment]
@anelda - These look great! Thanks for providing such a detailed write up of both the Africa Task Force activities and CarpentryConnect JoBurg.

@maneesha - I've left a few minor comments on the blog posts. Could you please incorporate those minor changes (typos), reset the posting dates, and separate the blog posts into two different PRs so that they can be merged separately. If we merge them together, they will post on the same day - even if the posting dates are different. There will be another post coming out today and tomorrow, so can you schedule these for Wednesday and Thursday?

**Comment by anelda** [comment]
@maneesha I haven't done anything about @ErinBecker's comments based on the way she posted them. If you want me to fix the typos, etc, please let me know. Don't want to duplicate it if you're already doing it.

PS: couldn't see the typos - only American vs English spelling but maybe I've got a blindspot...

Link to a very informal version of the minutes - https://pad.carpentries.org/2018-02-22-CarpentriesTaskForce

Please let me know if you need anything else from me?

File: _posts/2019/2019-01-11-african-task-force-update.md (hunk 3 above)
Comment (maneesha) [reply _posts/2019/2019-01-11-african-task-force-update.md]: @anelda   Thanks for sharing the link to the etherpad. However here you also say there's a GitHub repor for the meeting minutes.  Where is that?

Comment (anelda) [reply _posts/2019/2019-01-11-african-task-force-update.md]: @maneesha https://github.com/carpentries/african-task-force - we hardly used it, but here it is :-)

**Comment by anelda** [comment]
@maneesha @ErinBecker do you need me to do something about these posts to get them published? It would be great if we can get them out as we are getting a lot of requests for info about both CarpentryConnect and the TF. Please let me know what I can do to help? Thanks!

**Comment by maneesha** [comment]
My apologies @anelda !  We had a week-long, all-staff retreat last week and this slipped through the cracks.  I made a few edits and am merging now.

**Pull request merged**